examples/api/src/index.ts

```diff
@@ -113,12 +114,34 @@ async function asyncMain() {
     }
   ]
 
+  if (!process.env.MAILGUN_API_KEY) throw new Error('No MAILGUN_API_KEY defined in environment.')
+  if (!process.env.MAILGUN_BASE_URL) throw new Error('No MAILGUN_BASE_URL defined in environment.')
+  if (!process.env.MAILGUN_WEBHOOK_SECRET)
+    throw new Error('No MAILGUN_WEBHOOK_SECRET defined in environment.')
+
+  let mailProvider
+  if (
+    process.env.MAILGUN_API_KEY &&
+    process.env.MAILGUN_BASE_URL &&
+    process.env.MAILGUN_WEBHOOK_SECRET
+  ) {
+    mailProvider = new MailgunMailProvider({
+      id: 'mailgun',
+      name: 'Mailgun',
+      fromAddress: 'mails@wepublish.media',
```

Comment:
I guess we can add this as an env variable?

Reply:
It's just the examples/api. We don't need to over engineer that.